app/assets/stylesheets/base/form.scss

```diff
@@ -74,3 +81,17 @@
   text-transform: uppercase;
   margin: 10px;
 }
+
+.error-messages {
+  list-style-type:decimal;
```

Comment:
WHY?

Reply:
If you are using an `ul` tag, which is a unordered list, why list them in a decimal way? is make sense if it is an ordered list. Either way, in this case, make no sense be an ordered list.